rust/lance/src/dataset/scanner.rs

```diff
@@ -355,6 +361,15 @@ impl Scanner {
         Ok(self)
     }
 
+    pub fn full_text_search(&mut self, column: &str, query: &str) -> Result<&mut Self> {
```

Comment:
lets have a docstring for the format of the query?

IIRC, tantivy does not need to provide column. It covers all columns in the index.

Reply:
tantivy requires to specify the index when search, and have to specify the schema when creating index, so the index know which fields to search
btw, should we change this to `columns: IntoIter<AsStr>`?